src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -182,4 +187,62 @@ private EnumerateSnapshotAccessRequest convertToEnumerateModel(
     }
     return enumerateModel;
   }
+
+  // Hardcoded stubbed data, will remove once query is implemented.
+  private static SnapshotBuilderConcept createConcept(int id, String name, boolean hasChildren) {
+    return new SnapshotBuilderConcept().id(id).name(name).count(100).hasChildren(hasChildren);
+  }
+
+  private static final List<SnapshotBuilderConcept> CONCEPTS =
+      List.of(
+          createConcept(100, "Condition", true),
```

Comment:
This is a copy of the data from https://github.com/DataBiosphere/terra-ui/blob/4af250b77a41ac9f53372858ef9ed4ca7a661ed8/src/dataset-builder/TestConstants.ts#L135